src/billing/Project.ts

```diff
@@ -220,7 +269,7 @@ const ProjectDetail = ({
                   ]
                 : [
                     'This shared billing project has only one owner. Consider requesting ',
-                    h(Link, { mailto: projectOwners[0].email }, [projectOwners[0].email]),
+                    h(Link, { href: `mailto:${projectOwners[0].email}` }, [projectOwners[0].email]),
```

Comment:
This is a bug fix that TypeScript pointed out-- `mailto` has to be part of the href, else it doesn't work.